freecad/trails/geometry/line.py

```diff
@@ -235,7 +235,7 @@ def get_ortho_vector(line, distance, side=''):
     _left = Vector(-_delta.y, _delta.x, 0.0)
 
     _coord = get_coordinate(start, bearing, distance)
-
+    
```

Comment:
Something I work really hard to do is avoid is unnecessary whitespace.  I think most code editors can auto-remove whitespace for you...  also, I haven't checked, but if you use tab to indent, make sure it converts to spaces.  Mixing spaces and tabs can create a real mess.

Reply:
I will pay attention for next time :)